doc/source/changes/version_0_25.rst.inc

```diff
@@ -72,6 +72,36 @@
      a1   1   1   5
      a2  -4  -5   0
 
+* implemented Session.transpose(axes) to reorder axes of all arrays within a session, ignoring missing axes for each
+  array. Let us first create a test session and a small helper function to display sessions as a short summary.
+
+    >>> arr1 = ndtest((2, 2, 2))
+    >>> arr2 = ndtest((2, 2))
+    >>> sess = Session([('arr1', arr1), ('arr2', arr2)])
+    >>> def print_summary(s):
+    ...     print(s.summary("{name} -> {axes_names}"))
+    >>> print_summary(sess)
+    arr1 -> a, b, c
+    arr2 -> a, b
+
+  Transpose axes of all arrays
+
+    >>> print_summary(sess.transpose('b', 'a'))
```

Comment:
Why not just sess.transpose('b') ? 
Given the examples, I fear people will think that transpose works like swap (people not used to Numpy).